### PR TITLE
fix: Pfe nav & pfe-cta hotfixes

### PR DESCRIPTION
**Pull request by starryeyez024**
## Issue 1

Currently on very small mobile screens or regular mobile screens (iphone X) with zoom turned on, there is a horizontal scroll bar caused by the nav, which is too wide with the logo, the padding, the hamburger and the 9box icon.

This PR updates settings so that the `--pfe-navigation__logo--MinWidth` works as intended, and if sites have a large logo they can drop the size with a media query, like:

```scss
@media (max-width: 440px) {
  pfe-navigation {
    #{--pfe-navigation__logo--MinWidth}: 124px;
  }
}
```

## Issue 2

The pfe-cta specifies a 1px transparent border on the root of the document which is visible around the page; this removes that.
image

## Suggested testing: 

Play with the vars at the top of the demo/index.html file:
```
  <style>
  pfe-navigation {
     --pfe-navigation__logo--MinWidth: 280px;
  }    
  @media (max-width: 440px) {
    pfe-navigation {
       --pfe-navigation__logo--MinWidth: 160px;
    }
  }
  </style>
```

### CODE REVIEW

File: elements/pfe-navigation/src/pfe-navigation.scss [hunk 1]
```diff
@@ -79,6 +79,7 @@ pfe-icon {
 // Note: These styles do not pick up in Edge
 ::slotted([slot="logo"]) {
   margin: 0 !important;
+  max-width: 100%;
```
Comment (chrisdo1) [inline elements/pfe-navigation/src/pfe-navigation.scss]: Do we need this? Defaults to max-width: none, right?

File: elements/pfe-navigation/src/pfe-navigation.scss [hunk 2]
```diff
@@ -156,14 +158,15 @@ pfe-icon {
...
     align-items: flex-start;
     justify-content: center;
-    min-width: #{pfe-local($region: logo, $cssvar: MinWidth)};
+    width: #{pfe-local($region: logo, $cssvar: MinWidth)};
+    flex: 0 0 #{pfe-local($region: logo, $cssvar: MinWidth)};
     
     padding-top: #{pfe-local(Padding--vertical)};
     padding-bottom: #{pfe-local(Padding--vertical)};
-    padding-right: 48px; // ensure gap between main menu links and logo
+    padding-right: 25px; // ensure gap between main menu links and logo
```
Comment (chrisdo1) [inline elements/pfe-navigation/src/pfe-navigation.scss]: Is it helpful to use variables for these values?
`padding-right: 25px;`
becomes
`padding-right: calc( #{pfe-var(container-spacer)} + 9px );`

File: elements/pfe-navigation/src/pfe-navigation.scss [hunk 3]
```diff
@@ -156,14 +158,15 @@ pfe-icon {
...
 
-    @media screen and (min-width: pfe-breakpoint("lg-mobile", $map: false)) {
-        padding-right: 20px;
+    @media screen and (min-width: pfe-breakpoint("grid-breakpoint--md")) {
+        padding-right: 48px;
```
Comment (chrisdo1) [inline elements/pfe-navigation/src/pfe-navigation.scss]: Same here:
`padding-right: calc( #{pfe-var(container-padding)} * 3 );`

File: examples/index.html [hunk 4]
```diff
@@ -59,6 +59,7 @@ <h1>Demo pages</h1>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-health-index/demo">pfe-health-index</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-icon/demo">pfe-icon</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-icon-panel/demo">pfe-icon-panel</a></pfe-cta>
+				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-layouts/demo">pfe-layouts</a></pfe-cta>
```
Comment (castastrophe) [inline examples/index.html]: This is an old folder that no longer exists

Comment (starryeyez024) [reply examples/index.html]: this was auto-added when i compiled, how do I prevent that?

File: elements/pfe-navigation/src/pfe-navigation.scss [hunk 5]
```diff
@@ -156,14 +158,15 @@ pfe-icon {
     flex-shrink: 0; // Prevent flexbox from collapsing the padding.
     align-items: flex-start;
     justify-content: center;
-    min-width: #{pfe-local($region: logo, $cssvar: MinWidth)};
+    width: #{pfe-local($region: logo, $cssvar: MinWidth)};
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation.scss]:
```suggestion
    width: #{pfe-local($region: logo, $cssvar: MinWidth, $fallback: 155px)};
```

File: elements/pfe-navigation/demo/index.html [hunk 6]
```diff
@@ -42,11 +42,22 @@
...
+    <!-- emulate custom logo overrides -->
+  <style>
+  pfe-navigation {
+     --pfe-navigation__logo--MinWidth: 280px;
+  }    
+  @media (max-width: 440px) {
+    pfe-navigation {
+       --pfe-navigation__logo--MinWidth: 160px;
+    }
+  }
```
Comment (starryeyez024) [inline elements/pfe-navigation/demo/index.html]:
```suggestion
  }
    img {
    max-height: 100%;
  }
```

**Comment by starryeyez024** [comment]
@chrisdo1  moving this here from the webdrupal shame file, can you add this in? 
```
//Edge hack to fix height of mobile tray
// @TODO move to pfe navigation lightdom styles.
@media screen and (max-width: 1023px) {
  .pfe-navigation-item .pfe-navigation-item__tray {
    max-height: none !important;
  }
}
```

File: elements/pfe-navigation/src/pfe-navigation.scss (hunk 2 above)
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation.scss]: @starryeyez024 I made this change. Feel free to revert or change it.

File: elements/pfe-navigation/src/pfe-navigation.scss (hunk 3 above)
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation.scss]: @starryeyez024 I made this change.

File: examples/index.html (hunk 4 above)
Comment (chrisdo1) [reply examples/index.html]: @starryeyez024 I removed this

**Comment by starryeyez024** [comment]
image

**Pull request merged**